### ISSUE TITLE
Support RTL text in timeline

### ISSUE THREAD

**Issue by anyone00** [ajbura/cinny#717]
### Describe the bug

The direction of RTL text in timelines is not set correctly.

Currently when viewing an RTL chat:
![viewing an RTL chat in cinny](https://user-images.githubusercontent.com/41706133/183094838-7352aaf7-966f-4eeb-81e3-d5973cf7de7f.png)

### Reproduction

1. join #test-rtl:matrix.org
2. view the room timeline
3. text is left-to-right instead of right-to-left

### Expected behavior

the text direction should be right-to-left

### Platform and versions

```shell
1. OS: [Windows 10, Linux]
2. Browser: [chrome 105, firefox 103]
3. Cinny version: [v2.0.4 (dev.cinny.in)]
4. Matrix homeserver: [e.g. matrix.org]
```

**Comment by anyone00** [comment]
might be related to https://github.com/cinnyapp/cinny/pull/139

**Comment by anyone00** [comment]
Thanks!

**Comment by anyone00** [comment]
if you write a RTL message and include a code snippet in it. The code snippet also gets RTL direction styling.

When a code snippet is sent with LTR language:
![code snippet is sent with LTR language](https://user-images.githubusercontent.com/41706133/183689982-137bd378-0b18-427e-a259-0afc754c7424.png)

When a code snippet is sent with RTL language:
![code snippet is sent with RTL language](https://user-images.githubusercontent.com/41706133/183689516-88670e3a-ef6d-4377-af84-2ee32d768907.png)
